source/agora/test/Base.d

```diff
@@ -1231,6 +1231,13 @@ public class TestAPIManager
     {
         this.ensureTxInPool(iota(this.clients.length), hash);
     }
+
+    /// Ditto
+    public void ensureTxInPool (in Transaction tx)
+    {
+        iota(this.clients.length).each!(idx => this.clients[idx].postTransaction(tx));
+        this.ensureTxInPool(tx.hashFull());
+    }
```

Comment:
This was recently changed, so I guess bad rebase ?

Reply:
should've used `postAndEnsureTxInPool`